client/components/CheckoutCartSumm.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react'
+import CheckoutCartSummCard from './CheckoutCartSummCard'
+
+const CheckoutCartSumm = ({items}) => {
+  //const {orderItem} = items
+
+  const numberWithCommas = num => {
+    return num.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ',')
+  }
+
+  const orderTotal = itemsArr => {
+    let result = 0
+    itemsArr.forEach(item => {
+      result += item.price * item.orderItem.quantity
+    })
+    return result
+  }
```

Comment:
Could use an Array.reduce() method. Don't need to change it now, but could keep it in mind for optimizing later